### PR TITLE
add endpoint to retrieve movie crew

### PR DESCRIPTION
**Pull request by aladdin91**
# Description
add endpoint to retrieve all movie crew

# Jira task link
https://hackyourfuture-dk.atlassian.net/browse/VIEW-20

# Screenshots (if applicable)

### CODE REVIEW

File: packages/server/api/controllers/movies.controller.js [hunk 1]
```diff
@@ -24,6 +24,31 @@ const getMovieByID = async (id) => {
...
+        'full_name',
+        'birth_date',
+        'image_location',
+        'role',
+      )
+      .from('movie_crew')
+      .join('crew_members', 'movie_crew.crew_member_id', '=', 'crew_members.id')
+      .where('movie_id', id);
+    if (!crew || crew.length === 0) {
+      throw new Error(`No crew found for this movie`, 404);
```
Comment (itsgauravsaxena) [inline packages/server/api/controllers/movies.controller.js:44]: Throw error and catch in same block can lead to both throwing exception.
Have you tried to test this with an id that does not exist and see you can get expected error?

**Review by itsgauravsaxena — CHANGES_REQUESTED** [review]

File: packages/server/api/controllers/movies.controller.js (hunk 1 above)
Comment (aladdin91) [reply packages/server/api/controllers/movies.controller.js:44]: you are correct, i will send a message instead of throw error

Comment (itsgauravsaxena) [reply packages/server/api/controllers/movies.controller.js:61]: Maybe moving this error handling to router would be optimal.
You can refer to this PR https://github.com/HackYourFuture-CPH/CINEMANIA/pull/50/files#diff-f8f4871279607cea2f43141bd7d3600a2ead0fa38bb516e7d5be1eee4c308359

Comment (aladdin91) [reply packages/server/api/controllers/movies.controller.js]: i moved the error handling to router file and i will take the same method next time 👍

Comment (itsgauravsaxena) [reply packages/server/api/controllers/movies.controller.js]: Looks good. 
Could you move crew into a separate controller and router?

Comment (aladdin91) [reply packages/server/api/controllers/movies.controller.js]: i make a new separate controller and router for the cast, and waiting for the review 🙏

**Review by itsgauravsaxena — APPROVED** [review]

**Comment by aladdin91** [comment]
thanks a lot for the review 🙏, i will merge it now

**Pull request merged**